packages/math-input/src/components/keypad/__tests__/keypad-v2-mathquill.test.tsx

```diff
@@ -0,0 +1,185 @@
+import Color from "@khanacademy/wonder-blocks-color";
+import {Popover} from "@khanacademy/wonder-blocks-popover";
+import {render, screen} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import MathQuill from "mathquill";
+import * as React from "react";
+
+import "@testing-library/jest-dom";
+
+import Key from "../../../data/keys";
+import keyTranslator from "../../key-translator";
+import Keypad from "../index";
+
+type Props = {
+    onChangeMathInput: (mathInputTex: string) => void;
+};
+
+function V2KeypadWithMathquill(props: Props) {
+    const mathquillWrapperRef = React.useRef<HTMLDivElement>(null);
+    const [mathField, setMathField] = React.useState();
+
+    React.useEffect(() => {
+        if (!mathField && mathquillWrapperRef.current) {
+            const MQ = MathQuill.getInterface(2);
+            const mathFieldInstance = MQ.MathField(
+                mathquillWrapperRef.current,
+                {
+                    charsThatBreakOutOfSupSub: "+-*/=<>≠≤≥",
+                    handlers: {
+                        edit: () =>
+                            props.onChangeMathInput(mathFieldInstance.latex()),
```

Comment:
```suggestion
                        edit: (mathField) => {
                            props.onChangeMathInput(mathField.latex()),
                        }
```

Two things:
1. nit, non-blocking: unless the linter is forcing this style, I think it's best to wrap function bodies in brackets.
2. I think the edit callback provides the `mathField`, so that should be preferred.